README.md

```diff
@@ -31,12 +57,221 @@ const structure = [
 />;
 ```
 
-## Props
+ ## Material tree
+ 
+  <p align="center">
+      <img src="react-tree-demo.gif?raw=true" width="400" />
+  </p>
+  
+  ### Installation
+  
+   **Installation using npm:**
+  ```
+   npm install @kenshooui/material-tree --save
```

Comment:
You need react-tree as well as it's a peerDependency

Reply:
Also material core and icons (add a comment that they are needed if they don't exist in the project yet)